git_machete/git_operations.py

```diff
@@ -730,11 +737,17 @@ def matches(remote_branch: RemoteBranchShortName) -> bool:
         return matching_remote_branches[0] if len(matching_remote_branches) == 1 else None
 
     def get_merged_local_branches(self) -> List[LocalBranchShortName]:
-        return list(map(
-            lambda branch: LocalBranchFullName.of(branch).to_short_name(),
-            utils.get_non_empty_lines(
-                self._popen_git("for-each-ref", "--format=%(refname)", "--merged", "HEAD", "refs/heads"))
-        ))
+        if self.get_git_version() >= (2, 7, 6):  # earliest version of git to support 'for-each-ref --merged'
+            return list(
+                map(lambda branch: LocalBranchFullName.of(branch).to_short_name(),
+                    utils.get_non_empty_lines(
+                        self._popen_git("for-each-ref", "--format=%(refname)", "--merged", "HEAD", "refs/heads"))))
+        else:
+            return list(
+                filter(lambda branch: self.is_ancestor_or_equal(branch, self.get_commit_sha_by_revision(AnyRevision.of('HEAD'))),
```

Comment:
`get_head`?

Reply:
Or even:

```suggestion
                filter(lambda branch: self.is_ancestor_or_equal(branch, 'HEAD')),
```

and then `get_head` isn't needed (?)

Reply:
Yes, indeed `get_head()` isn't needed.